Modules/socketmodule.c

```diff
@@ -2045,14 +2045,21 @@ getsockaddrarg(PySocketSockObject *s, PyObject *args,
             memset(addr, 0, sizeof(struct sockaddr_l2));
             _BT_L2_MEMB(addr, family) = AF_BLUETOOTH;
             _BT_L2_MEMB(addr, bdaddr_type) = BDADDR_BREDR;
+            int psm;
+            int cid = _BT_L2_MEMB(addr, cid);
+            unsigned char bdaddr_type = _BT_L2_MEMB(addr, bdaddr_type);
```

Comment:
Didn't we `bdaddr_type` just set to `BDADDR_BREDR` above?

Reply:
> Didn't we bdaddr_type just set to BDADDR_BREDR above?

Oh, I didn't notice. I simplified the code (twice).